### PR TITLE
reamde: split using project into categories

### PR DESCRIPTION
**Pull request by kopiczko**
This is the order in which we need to do vendor updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,7 +180,6 @@ the `DefaultKubeConfig` constant in the `giantswarm/e2e-harness/pkg/harness` pac
 - https://github.com/giantswarm/kubernetes-node-health
 - https://github.com/giantswarm/kvm-operator
 - https://github.com/giantswarm/net-exporter
-- https://github.com/giantswarm/operatorkit
 - https://github.com/giantswarm/release-operator
```
Comment (kopiczko) [inline README.md]: I just ran into issues where I need to do vendor updates in order. So I thought splitting projects into categories makes sense here.

**Review by xh3b4sd — APPROVED** [review]
Nice. Do you think it would make sense to write a sentence or two about the development process in the specific readmes?

**Review by MarcelMue — APPROVED** [review]
Thanks for the effort <3

**Comment by kopiczko** [comment]
> Do you think it would make sense to write a sentence or two about the development process in the specific readmes?

I can't see how it could be not good, but let's maybe just paste link to some description in fmt to keep it DRY.

**Pull request merged**